ci/scenarios/vDisk/validate_clone_disk_test/main.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (C) 2016 iNuron NV
+#
+# This file is part of Open vStorage Open Source Edition (OSE),
+# as available from
+#
+#      http://www.openvstorage.org and
+#      http://www.openvstorage.com.
+#
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+# as published by the Free Software Foundation, in version 3 as it comes
+# in the LICENSE.txt file of the Open vStorage OSE distribution.
+#
+# Open vStorage is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY of any kind.
+
+import json
+import time
+from ci.main import CONFIG_LOC
+from ci.helpers.api import OVSClient
+from ci.setup.vdisk import VDiskSetup
+from ci.helpers.vpool import VPoolHelper
+from ci.remove.vdisk import VDiskRemover
+from ovs.log.log_handler import LogHandler
+from ci.validate.vdisk import VDiskValidation
+
+
+class VDiskCloneChecks(object):
+
+    CASE_TYPE = 'FUNCTIONAL'
+    LOGGER = LogHandler.get(source="scenario", name="ci_scenario_vdisk_clone")
+    PREFIX = "integration-tests-clone-"
+    VDISK_SIZE = 10737418240  # 10GB
+    CLONE_CREATE_TIMEOUT = 180
```

Comment:
ok - but I'd still prefer readability over 75 us:
In [13]: x = time.time(); y = 1; z = time.time() - x

In [14]: x = time.time(); y = 10 * 1024 ** 3; z1 = time.time() - x 

In [15]: z1 - z
Out[15]: 7.510185241699219e-05